openlibrary/plugins/upstream/tests/test_account.py

```diff
@@ -21,6 +33,120 @@ def test_create_list_doc(wildcard):
         "expires_on": wildcard
     }
 
+
+def test_process_goodreads_csv():
+    with open_test_data('goodreads_library_export.csv') as reader:
```

Comment:
Is this a [context manager]() or is it leaving the file open until the process is restarted?
https://docs.python.org/3/library/contextlib.html

`with open()` is done to automatically close the file when the `if` block ends.

Reply:
I was just doing that so it would automatically close the file when the block ends.